src/Simulation.jl

```diff
@@ -246,6 +246,69 @@ export getadjointsensitivities
     return sensspcs,sensrxns,sensspcnames,senstooriginspcind,senstooriginrxnind
 end
 
+@inline function getsensdomain(domain::ConstantTPDomain,ind::Int64)
```

Comment:
Is there a way for us to generalize this to one function that works for every domain and doesn't need updated everytime we add a domain? It can certainly done using Metaprogramming, there might be a slightly better way though.